app/helpers/application_helper.rb

```diff
@@ -368,26 +368,6 @@ def action_buttons(*args)
     end
   end
 
-  def avatar_image_tag(user, html_options = {})
-    if user.avatar_hash.present?
-      image_tag("avatars/#{user.avatar_hash}.jpg", html_options)
```

Comment:
The avatar_hash is actually coming from `ldap` and there is probably no reason removing this path.